tests/testthat/test-carehomes.R

```diff
@@ -219,3 +227,37 @@ test_that("can run the particle filter on the model 2", {
   ll2 <- pf2$run(pars)
   expect_lt(abs(ll1 - ll2), 60)
 })
+
+
+test_that("Test cases trajectories by age add up", {
```

Comment:
This test probably would be better place in test-carehomes-check

Reply:
Also would therefore write it to be more in-line with the tests in there, see e.g. the "tots all summed correctly" test

Reply:
moved